doc/buildAndProgram.md

```diff
@@ -14,7 +14,7 @@ To build this project, you'll need:
  - A reasonably recent version of CMake (I use 3.16.5)
  - lv_font_conv, to generate the font .c files
    - see [lv_font_conv](https://github.com/lvgl/lv_font_conv#install-the-script)
-   - install npm (commonly done via the package manager)
+   - install npm (commonly done via the package manager. ensure node's version is at least 12)
```

Comment:
A dot `.` inside a bracketed comment feels wrong, use a comma `,` instead

```suggestion
   - install npm (commonly done via the package manager, ensure node's version is at least 12)
```

Reply:
it kind of looks like part of the installation step, which it really isn't. but it does look weird with the dot,
and moving it out just takes too much space for something so basic.